linera-execution/src/execution_state_actor.rs

```diff
@@ -112,14 +112,7 @@ where
             }
 
             OwnerBalances { callback } => {
-                let mut balances = Vec::new();
-                self.system
-                    .balances
-                    .for_each_index_value(|owner, balance| {
-                        balances.push((owner, balance));
-                        Ok(())
-                    })
-                    .await?;
+                let balances = self.system.balances.index_values().await?;
```

Comment:
Isn't it called `.iter()` in the standard Rust library?

Reply:
I was about to make the same comment, but I think this one returns the whole `Vec` at once, rather than an iterator.